posthog/temporal/data_imports/pipelines/helpers.py

```diff
@@ -5,19 +5,13 @@
 CHUNK_SIZE = 10_000
 
 
-async def check_limit(
+async def is_job_cancelled(
     team_id: int,
     job_id: str,
-    new_count: int,
-):
+) -> bool:
     model = await aget_external_data_job(team_id, job_id)
 
-    if new_count >= CHUNK_SIZE:
-        new_count = 0
```

Comment:
This wasn't really doing anything before from what I can tell - so best to just remove it